src/tespy/components/heat_exchangers.py

```diff
@@ -867,6 +867,26 @@ def calc_parameters(self):
 
         self.check_parameter_bounds()
 
+    def exergy_balance(self, Tamb):
+        r"""
+        Perform exergy balance of a simple heat exchanger.
+
+            .. math::
+
+                T_m = \frac{T_{ein} - T_{aus}}{\ln\left({\frac{T_{ein}}
+                                                         {T_{aus}}}\right)}
+                Ex_{output} = \dot{m}_{in} \cdot (ex_{ph,out} - ex_{ph,in})
+                Ex_{input} = \left(1 - \frac{T_{amb}}
+                                   {T_m}\right) \cdot \dot{Q}
+                Ex_{loss} = 0
```

Comment:
These lines do not get linebreaks in LaTeX, we need to add extra newlines or "\\" at the end of a line (see https://tespy--215.org.readthedocs.build/en/215/api/tespy.components.html#tespy.components.heat_exchangers.condenser.exergy_balance).